e2e/container/run_test.go

```diff
@@ -17,8 +16,8 @@ const registryPrefix = "registry:5000"
 func TestRunAttachedFromRemoteImageAndRemove(t *testing.T) {
 	image := createRemoteImage(t)
 
-	result := icmd.RunCmd(shell(t,
-		"docker run --rm %s echo this is output", image))
+	result := icmd.RunCommand("docker", "run", "--rm", image,
+		"echo", "this", "is", "output")
```

Comment:
Should "this is output" be a single string here?

Reply:
Both work :stuck_out_tongue_winking_eye: